src/client/helpers/api/api.client.js

```diff
@@ -19,17 +26,79 @@ const throwApiError = ({ data = {}, status = 500 }) => {
   throw new ApiError(data, status);
 };
 
+const refreshToken = () => api.post('/account/refresh-token');
+
+const isOkStatus = status => status >= 200 && status < 300;
+
+const refreshTokenIfNeeded = async () => {
+  if (getAccessToken()) {
+    return;
+  }
+
+  const tokenResponse = await refreshToken();
+
+  if (!isOkStatus(tokenResponse.status)) {
+    throwApiError({ data: { errors: generalError }, status: tokenResponse.status });
+  }
+};
+
+const redirectToLogin = () => {
+  window.location.href = config.landingLoginUrl;
+};
+
+const handleResponse = async (response, skipUnauthorized) => {
```

Comment:
Please expose `skipUnauthorized` as a named parameter with default = false, as it's not clear from the call, what the purpose of the parameter 

So the call will look like: `await handleResponse(await api(options), { skipUnauthorized: true });`